packages/docs/pages/app/datasources.mdx

```diff
@@ -174,7 +174,8 @@ const growthbook = new GrowthBook({
   trackingCallback: function(experiment, result) {
     mixpanel.track("$experiment_started", {
       "Experiment name": experiment.key,
-      "Variant name":  result.variationId
+      "Variant name":  result.variationId,
+      "$source": "growthbook",
     })
```

Comment:
We show this same code a couple places on the front-end as well that we should add $source to (e.g. in the code snippet modal and on the data source page).  If you do a search for `mixpanel.track` in the codebase you should find it.